### DISCUSSION TITLE
Viewing server as a role

### DISCUSSION THREAD

**Discussion by vico93** [stoatchat/discussions#588] [Feature Suggestions]
Be able to test server roles by simulating how the server would present to them would be a huge benefit for server owners. I hope this feature from Discord could be emulated by Revolt as well.

[Reference for how Discord implements this](https://support.discord.com/hc/en-us/articles/360055709773-View-as-Role-FAQ).

**Comment by treminaor** [comment]
Yeah this would be a nice feature. My way around this right now is to use a second account on another email, invite that second account to the server and then they are yours to test different roles on. That's how you used to do it in Discord before they implemented that nifty feature.
